criu/seize.c

```diff
@@ -24,6 +24,46 @@
 #include "xmalloc.h"
 #include "util.h"
 
+char *task_comm_info(pid_t pid, char *comm, size_t size)
+{
+	int ret = 0;
```

Comment:
It looks more reasonable to initialize it with -1.

Reply:
Reworked it in:
https://github.com/checkpoint-restore/criu/pull/2095